.packit.yaml

```
@@ -0,0 +1,29 @@
+specfile_path: ansible-lint.spec
+actions:
+  create-archive:
+    - "python3 setup.py sdist --dist-dir ."
```

Let's not call this directly. If we let some envs bypass PEP 517, it just adds more maintenance burden in the end. Also, you basically don't run the same thing that is checked in the CI potentially making the output different w/o realizing that.
```suggestion
    - python3 -m pip install build --user
    - python3 -m build --sdist --outdir .
```